README.md

```diff
@@ -46,6 +46,7 @@ A registry of tutorials, workshop material, videos and other educational resourc
 
 | Title | Presenter | Video | Content | Binder |
 | :--- | :---: | :---: | :---: | :---: |
+| Data Management with DataLad, UKE Hamburg | Adina Wagner, Michał Szczepanik |  [:movie_camera:](https://youtube.com/playlist?list=PLEQHbPfpVqU5RSPiyFuPdDlSUEd-XoPV-) | |
```

Comment:
```suggestion
| Data Management with DataLad, UKE Hamburg | Adina Wagner, Michał Szczepanik |  [:movie_camera:](https://youtube.com/playlist?list=PLEQHbPfpVqU5RSPiyFuPdDlSUEd-XoPV-) | | |
```

didn't check this out locally to test, but just adding the extra pipe character in case its absence breaks rendering